src/main/java/com/Acrobot/ChestShop/Events/Economy/CurrencyTransferEvent.java

```diff
@@ -150,6 +157,13 @@ public Direction getDirection() {
         return direction;
     }
 
+    /**
+     * Gets the {@link TransactionEvent} associated with this currency transfer event.
+     *
+     * @return the transaction event.
+     */
+    public TransactionEvent getTransactionEvent() { return transactionEvent; }
```

Comment:
Small nitpick but that doesn't match the rest of the code. (Wont block the review though, either you change it or I will myself before merging)

Reply:
Weird. On my IDE all of them are collapsed :p